spec/request/api/v1/book_spec.rb

```diff
@@ -0,0 +1,11 @@
+require 'spec_helper'
+
+describe Api::V1::BooksController, type: :controller do
```

Comment:
There are two tests for the controllers, one should be for the model and another for the controller.